fixtures/allow_additional_props.json

```diff
@@ -162,9 +162,9 @@
         "mult": {
           "type": "number",
           "enum": [
-            1,
+            1.0,
```

Comment:
these are just result of running `go test -update ./...`

Reply:
I'm not happy about those... these slipped the net. Will try and figure out what happened there.

Reply:
Okay, the new output is correct and aligned with expectations. The main branch is now up to date which should simplify this PR.